### DISCUSSION TITLE
Laravel OTP Manager

### DISCUSSION THREAD

**Discussion by salehhashemi1992** [HassanZahirnia/laravel-package-ocean#146] [Package Suggestions]
Generate, send, verify, and manage one-time passwords for Laravel

Features
- Generate OTP codes
- Send OTPs via mobile numbers
- Resend OTPs with built-in throttling
- Verify OTP codes
- Supports multiple types of OTPs (e.g., login, reset password)

**Comment by HassanZahirnia** [comment]
Hey, is that a package suggestion or a package idea? I don't see any links.

**Reply by salehhashemi1992** [reply]
oh. excuse me.
this is the link:
https://github.com/salehhashemi1992/laravel-otp-manager

**Reply by salehhashemi1992** [reply]
@HassanZahirnia

**Reply by HassanZahirnia** [reply]
Thanks for the suggestion 😊 I've added the package!
